packages/core/src/index.ts

```diff
@@ -43,14 +43,14 @@ export { DeprecationNotice } from './notices/DeprecationNotice.js';
  * Export events
  */
 export { TapEvent } from './events/TapEvent.js';
-export type { FocusedChangedEvent } from './types/events';
+export type { FocusedChangedEvent } from './types/events.js';
 
 /**
  * Export common interfaces
  */
-export type { MultiValue } from './interfaces/MultiValue';
-export type { StyleMap } from './interfaces/StyleMap';
-export type { StyleInfo } from './interfaces/StyleInfo';
```

Comment:
@goremikins I remember you had mentioned the js extension for import `type` when we migrated Lit v2, and what is the problem why we should not add it. Can you tell me again?

Reply:
`js` files are consumed by browser, while types by TypeScript compiler. Therefore it does not make logical sense to include `js` extension. Another reason, consider these scenario:
```
// types.ts
type A = {};
export A;
```
when `types.ts` is complied, `types.js` would look like:
```
export {};
```

Reply:
You are right. When you click on `types.js` import path it will still link to `types.d.ts` correctly because names are the same, but I think it might be possible to ignore `import type` specifically.